pom.xml

```diff
@@ -50,6 +54,12 @@
             <artifactId>junit-jupiter-engine</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jetbrains</groupId>
+            <artifactId>annotations</artifactId>
+            <version>24.0.1</version>
+            <scope>compile</scope>
```

Comment:
Minor feedback - Compile scope is the default - shouldn't need to specify it unless needing to override what the maven parent specifies.

```suggestion
```